### DISCUSSION TITLE
Show the number of creatures on the map in numbers

### DISCUSSION THREAD

**Discussion by DM-VLK** [ihhub/fheroes2#7593] [General]
### Preliminary checks

- [X] I've checked that there aren't [**other open issues**](https://github.com/ihhub/fheroes2/issues?q=is%3Aissue) on the same topic.

### Describe the problem requiring a solution

Sometimes you can forget how many creatures correspond to a word. I propose to additionally write the number of creatures in the squad on the map in numbers.

### Describe the possible solution

Something like this:
![4](https://user-images.githubusercontent.com/112470261/214880780-cab19aa9-ee62-4300-b041-ca1aebeddb24.png)

**Comment by ihhub** [comment]
I am moving this topic to the discussion section. We had similar talks in the past.
